### PR TITLE
Enum classify

### PR DESCRIPTION
**Pull request by rmpowell77**
Converting enums to enum classes.  This has better scope properties.  where enum exposes the symbol name external to the enum scope (outside the braces), enum class restricts the symbol name internally to the scope. This results in less symbol pollution.

### CODE REVIEW

File: src/core/print_ps.cpp [hunk 1]
```diff
@@ -564,7 +564,7 @@ void PrintShowToPS::PrintContSections(std::ostream& buffer, const CC_sheet& shee
 	for (auto& text : continuity)
 	{
 		if (!text.GetOnSheet()) continue;
-		gen_cont_line(buffer, text, PSFONT_NORM, this_size);
+		gen_cont_line(buffer, text, PSFONT_TYPE::NORM, this_size);
```
Comment (noahsark769) [inline src/core/print_ps.cpp]: Just wondering, why are PSFONT_TYPE and SYMBOL_TYPE uppercase snake case while everything else seems to me capital camel case?

**Comment by kmdurand** [comment]
I really like this, but Visual Studio is pretty dumb:
http://msdn.microsoft.com/en-us/library/hh567368.aspx
They don't support constexpr or noexcept...

**Comment by rmpowell77** [comment]
What!?  No support for constexpr and noexcept?  That's so lame

**Comment by rmpowell77** [comment]
We will have to hold off on this submission till we determine what visual studio can support

**Comment by rmpowell77** [comment]
This change needs to be redone.  Going to set as close

**Pull request closed without merging**